CONTRIBUTING.md

```diff
@@ -51,36 +51,42 @@ Included in this repo is the file `helphub.wordpress.2017-06-15.xml`. Import thi
 
 Once you have local install of HelpHub up and running, you can contribute with pull requests either from your own fork or after you've added as a contributor directly in this repository. We are using [Travis CL](https://travis-ci.org/) for tests on every pull request. You can, also, run these tests localy before pushing your code (more on this later). Development covers work on both, theme and plugins and requires following [best practices](https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/) and [WordPress Coding Standards](https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards).
```

Comment:
/localy/locally/ :)

Reply:
Thanks, fixed in https://github.com/Kenshino/HelpHub/pull/133/commits/98247d42816d79d79f2b7c4c27d8fe85ef1193a0